### PR TITLE
feat: customize base token for each EVM network

### PR DESCRIPTION
**Pull request by Sekhmet**
Closes: https://github.com/snapshot-labs/sx-ui/issues/723

Depends on: https://github.com/snapshot-labs/stamp/pull/88

## Test plan
- Run local stamp for logo support: https://github.com/snapshot-labs/stamp/pull/88
- Check http://localhost:8080/#/matic:0xa98d476118af1a2c9eab803db9688e33cf68435f/treasury

## Screenshots

image

### CODE REVIEW

**Comment by bonustrack** [comment]
This wont work for DAOs on Arbitrum as the base token is ETH, now it show Arbitrum as token:
`http://localhost:8080/#/arb1:0x2fb231a3ac7c40888d2e6773f5600851b0c20ce4/treasury`. Also it would be better to move the logic to resolve token images to Stamp.

**Comment by Sekhmet** [comment]
@bonustrack moved it to Stamp.

File: src/views/Space/Treasury.vue [hunk 1]
```diff
@@ -175,7 +175,12 @@ watchEffect(() => {
             class="mx-4 py-3 border-b flex"
           >
             <div class="flex-auto flex items-center min-w-0">
-              <Stamp :id="asset.contractAddress" type="token" :size="32" />
+              <Stamp
+                :id="`eip155:${treasury.network}:${asset.contractAddress}`"
```
Comment (bonustrack) [inline src/views/Space/Treasury.vue:179]: @Sekhmet Wouldn't it be simpler to just use network short name instead of eip155 name?

**Review by bonustrack — APPROVED** [review]
tACK

**Review by bonustrack — CHANGES_REQUESTED** [review]
Actually just realized but we also have token in the "Send token" modal, would be great to do same change.

File: src/composables/useTreasury.ts [hunk 2]
```diff
@@ -19,6 +19,7 @@ export function useTreasury(spaceRef: Ref<NullableSpace>) {
     if (!chainId || !wallet) return null;
 
     return {
+      shortName: networkId,
```
Comment (bonustrack) [inline src/composables/useTreasury.ts:22]: We could keep `networkId` here to be consistent with others files 
We could 
```suggestion
     networkId,
```

**Review by bonustrack — APPROVED** [review]

**Pull request merged**